kaspresso/src/main/kotlin/com/kaspersky/kaspresso/kaspresso/Kaspresso.kt

```diff
@@ -109,66 +109,135 @@ data class Kaspresso(
-                    )
-
-                    atomWatcherInterceptors = mutableListOf(
-                        LoggingAtomWatcherInterceptor(libLogger)
-                    )
-
-                    webAssertionWatcherInterceptors = mutableListOf(
-                        LoggingWebAssertionWatcherInterceptor(libLogger)
-                    )
-
-                    viewBehaviorInterceptors = mutableListOf(
-                        AutoScrollViewBehaviorInterceptor(autoScrollParams, libLogger),
-                        SystemDialogSafetyViewBehaviorInterceptor(libLogger, uiDevice, adbServer),
-                        FlakySafeViewBehaviorInterceptor(flakySafetyParams, libLogger),
-                        FailureLoggingViewBehaviorInterceptor(libLogger)
-                    )
+            @Deprecated("Use `advanced()` builder.", ReplaceWith("Kaspresso.Builder.advanced(customize)"))
+            fun default(customize: Builder.() -> Unit = {}): Builder = advanced(customize)
 
-                    dataBehaviorInterceptors = mutableListOf(
-                        SystemDialogSafetyDataBehaviorInterceptor(libLogger, uiDevice, adbServer),
-                        FlakySafeDataBehaviorInterceptor(flakySafetyParams, libLogger),
-                        FailureLoggingDataBehaviorInterceptor(libLogger)
-                    )
+            /**
+             * Simple (preconfigured with logging and flaky-safe features) [Builder] for test environment configuration.
+             *
+             * If you desire to override some variables in Configurator please use [customize] parameter here.
+             *
+             * The example is:
+             * ```
+             * class ComposeTest : TestCase(
+             *     kaspressoBuilder = Kaspresso.Builder.default {
+             *         screenshots = ScreenshotsImpl(libLogger, activities, "my_directory")
+             *     }
+             * )
+             * ```
+             * In this case your implementation of ```screenshots``` will be used for initializing of all other variables.
+             * So, your implementation has a priority over the default implementation while using [customize] parameter for the overriding.
+             *
+             * Otherwise, if you don't set custom implementation then default implementation of ```screenshots``` will be explored.
+             *
+             * Please, be aware, that overridings like:
+             * ```
+             * class ComposeTest : TestCase(
+             *     kaspressoBuilder = Kaspresso.Builder.default().apply {
+             *         screenshots = ScreenshotsImpl(libLogger, activities, "my_directory")
+             *     }
+             * )
+             * ```
+             * don't give you desired effect! Yes, you have an possibility to use your custom ```screenshots```, but
+             * in all interceptors and other variables default implementation of ```screenshots``` will be exploring.
```

Comment:
This description is confused because simple configuration does not have screenshot interceptors.